activesupport/lib/active_support/file_evented_update_checker.rb

```diff
@@ -0,0 +1,71 @@
+require 'listen'
+  module ActiveSupport
+    class FileEventedUpdateChecker
+      attr_reader :listener
+      def initialize(files, dirs={}, &block)
+      	@files = Set.new
+      	@files = files.map { |f| File.expand_path(f)}
+      	@dirs = Hash.new
+      	dirs.map do |key,value|
+          @dirs[File.expand_path(key)] = Array(value) if !Array(value).empty?
+      	end
+      	@block = block
+      	@modified = false
+      	@listener = Listen.to(*base_directories, relative_paths: false)
```

Comment:
Is `relative_paths` working? Grepping the listen project for "relative_paths" gives no occurrence. The README documents a `relative` option though. Maybe it's related to the old version in the Gemfile?